pkg/webhook/server/webhook_cert_init.go

```diff
@@ -32,7 +34,10 @@ func InitWebhookServer(ctx context.Context) error {
 	}
 
 	ns, err := k8sutil.GetOperatorNamespace()
-	if err != nil {
+	if err == k8sutil.ErrRunLocal{
+		ns = os.Getenv("WATCH_NAMESPACE")
+		log.Info(fmt.Sprintf("Running operator in local mode; watching namespace %s", config.ConfigMapReference.Namespace))
```

Comment:
Should not we early fail here with a message like - `Operator is run in local mode; webhooks are not supported. Please disable webhooks and rerun operator.`
![Screenshot_20200525_114053](https://user-images.githubusercontent.com/5887312/82795424-adf6fe00-9e7c-11ea-8a40-fdd8feaeecf0.png)
?
What is the purpose to create/update service and configmap?

Reply:
Mostly that switching between local and in-cluster development requires updating the configmap, making it tedious to switch between testing something on the cluster and starting up a local instance for debugging. If you're running locally, you don't care about webhooks because they'll never work, and if you're running in-cluster the operator has to make the services consistent anyways.

Probably the best solution is to just assume webhooks are disabled when running locally.

Reply:
Then again, I used local debugging to step through the service creation process, so it's still useful even if webhooks end up not initialized.